target/linux/ramips/dts/OMEGA2.dts

```diff
@@ -0,0 +1,188 @@
+/dts-v1/;
+
+#include "mt7628an.dtsi"
+
```

Comment:
Please include  <[dt-bindings/gpio/gpio.h](http://lxr.free-electrons.com/source/include/dt-bindings/gpio/gpio.h?v=4.4)>  and <[dt-bindings/input/input.h](http://lxr.free-electrons.com/source/include/dt-bindings/input/linux-event-codes.h?v=4.4)> here as well.

Use the GPIO_ACTIVE_LOW and GPIO_ACTIVE_HIGH macros afterwards in stead of 1 and 0 in the gpio parameters.

Use the Key macros instead of the hex values in linux,code.

Check the recent [ramips board additions](https://github.com/lede-project/source/commits/master/target/linux/ramips) for examples. The same applies to the OMEGA2p dts also.